airbyte-integrations/connectors/destination-rabbitmq/destination_rabbitmq/destination.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+
+
+from typing import Mapping, Any, Iterable
+
+from airbyte_cdk import AirbyteLogger
+from airbyte_cdk.destinations import Destination
+from airbyte_cdk.models import AirbyteConnectionStatus, ConfiguredAirbyteCatalog, AirbyteMessage, Status, Type
+
+import json
+import pika
+from pika.adapters.blocking_connection import BlockingConnection
+from pika.spec import BasicProperties
+from pika.credentials import PlainCredentials
+
+_DEFAULT_PORT = 5672
+
+
+def create_connection(config: Mapping[str, Any]) -> BlockingConnection:
+    username = config.get('username')
+    password = config.get('password')
+    virtual_host = config.get('virtual_host')
+    params = pika.ConnectionParameters(
+        host=config['host'],
+        port=config.get('port') or _DEFAULT_PORT
+    )
+    if username and password:
+        params.credentials = PlainCredentials(username=username, password=password)
+    if virtual_host:
+        params.virtual_host = virtual_host
+    return BlockingConnection(params)
+
+
+class DestinationRabbitmq(Destination):
+    def write(
+            self,
+            config: Mapping[str, Any],
+            configured_catalog: ConfiguredAirbyteCatalog,
```

Comment:
you're not using the `configured_catalog` take a look in the kvdb destination how to use this.

Reply:
Updated in b3f2cee to use catalog. Thanks for the heads up!